packages/repository-tests/src/crud/relations/acceptance/has-many-inclusion-resolver.relation.acceptance.ts

```diff
@@ -0,0 +1,273 @@
+            customerId: c1.id,
+            description: 'order from c1',
+          });
+          const result = await customerRepo.find({
+            include: [{relation: 'orders'}],
+          });
+
+          const expected = {
+            id: c1.id,
+            name: 'c1',
+            orders: [
+              {
+                id: o1.id,
+                description: 'order from c1',
+                customerId: c1.id,
+                isShipped: features.emptyValue,
+                // eslint-disable-next-line @typescript-eslint/camelcase
+                shipment_id: features.emptyValue,
+              },
+            ],
+            parentId: features.emptyValue,
+          };
+          expect(toJSON(result)).to.deepEqual([toJSON(expected)]);
+        });
+
+        it('returns related instances to target models via find() method', async () => {
+          const c1 = await customerRepo.create({name: 'Thor'});
+          const c2 = await customerRepo.create({name: 'Hella'});
+          const o1 = await orderRepo.create({
+            customerId: c1.id,
+            description: 'Mjolnir',
+          });
+          const o2 = await orderRepo.create({
+            customerId: c1.id,
+            description: 'Pizza',
+          });
+          const o3 = await orderRepo.create({
+            customerId: c2.id,
+            description: 'Blade',
+          });
+
+          const result = await customerRepo.find({
+            include: [{relation: 'orders'}],
+          });
+
+          const expected = [
+            {
+              id: c1.id,
+              name: 'Thor',
+              orders: [
+                {
+                  id: o1.id,
+                  description: 'Mjolnir',
+                  customerId: c1.id,
+                  isShipped: features.emptyValue,
+                  // eslint-disable-next-line @typescript-eslint/camelcase
+                  shipment_id: features.emptyValue,
+                },
+                {
+                  id: o2.id,
+                  description: 'Pizza',
+                  customerId: c1.id,
+                  isShipped: features.emptyValue,
+                  // eslint-disable-next-line @typescript-eslint/camelcase
+                  shipment_id: features.emptyValue,
+                },
+              ],
+              parentId: features.emptyValue,
+            },
+            {
+              id: c2.id,
+              name: 'Hella',
+              orders: [
+                {
+                  id: o3.id,
+                  description: 'Blade',
+                  customerId: c2.id,
+                  isShipped: features.emptyValue,
+                  // eslint-disable-next-line @typescript-eslint/camelcase
+                  shipment_id: features.emptyValue,
+                },
+              ],
+              parentId: features.emptyValue,
+            },
+          ];
+          expect(toJSON(result)).to.deepEqual(toJSON(expected));
+        });
+
```

Comment:
I am missing the following test case for `find` method:

- Create two customers
- Create one order for each customer
- Run `find` with a filter limiting the results to only one of the customers
- Verify that data from the other customer (customer, order) did not leak into the results

Reply:
I test this case in that `findById()` test case. Checks the result has correct instance and its related models. 
Since I don't want the test cases in this file look like testing `find*` methods, I will reword the titles to show the intention of each test case.

Reply:
I test this case in that findById() test case. Checks the result has correct instance and its related models.
Since I don't want the test cases in this file look like testing find* methods, I will reword the titles to show the intention of each test case.